README.md

```diff
@@ -305,6 +305,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 * [Mink](http://mink.behat.org/en/latest/) - Web acceptance testing.
 * [Mockery](https://github.com/mockery/mockery) - A mock object library for testing.
 * [ParaTest](https://github.com/paratestphp/paratest) - A parallel testing library for PHPUnit.
+* [Pest](https://pestphp.com/) - A Testing Framework with a focus on simplicity. It was carefully crafted to bring the joy of testing to PHP.
```

Comment:
I would suggest that we only use the first part of the sentence to:
- Better match the length of neighbouring items
- Remove the subjective opinion, that there's a general lack of joy, in the current ecosystem.
```suggestion
* [Pest](https://pestphp.com/) - A Testing Framework with a focus on simplicity.
```

Reply:
Thanks. I'll remember it.